Loops/sloveEquation.php

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Giải phương trình bậc hai</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+
+<body>
+    <?php 
+    $result="";
+    if($_SERVER['REQUEST_METHOD']=='POST'){
+        $resultA=isset($_POST['a'])?$_POST['a']:"";
+        $resultB=isset($_POST['b'])?$_POST['b']:"";
+        $resultC=isset($_POST['c'])?$_POST['c']:"";
+
+        $a=(int)$resultA;
+        $b=(int)$resultB;
+        $c=(int)$resultC;
+        // Deleta
+        $delta=($b*$b) - (4*$a*$c);
+        var_dump($delta);
+        if ($delta<0) {
+            $result='Phương trình vô nghiệm';
+        }
+        else if ($delta==0) {
```

Comment:
```suggestion
        } elseif ($delta == 0) {
```
chuẩn convention là như này